src/settings/appsettings.cpp

```diff
@@ -986,6 +986,33 @@ bool AppSettings::defaultShowMagnifier()
     return false;
 }
 
+QMap<QString, QVariant> AppSettings::tesseractParameters() const
+{
+    QMap<QString, QVariant> parameters;
+    m_settings->beginGroup("Tesseract");
+    QStringList keys = m_settings->childKeys();
+    for (const auto &key : keys) {
+        parameters.insert(key, m_settings->value(key));
+    }
```

Comment:
I prefer direct initialization to avoid unneeded extra things in scope. Also I use exact type when it increase readability.
```suggestion
    for (const QString &key : m_settings->childKeys())
        parameters.insert(key, m_settings->value(key));
```